pmd-cs/src/main/antlr4/net/sourceforge/pmd/lang/cs/antlr4/CSharpLexer.g4

```diff
@@ -0,0 +1,1105 @@
+// Eclipse Public License - v 1.0, http://www.eclipse.org/legal/epl-v10.html
```

Comment:
Can we simply use this? According to the faq https://www.eclipse.org/legal/epl-2.0/faq.php#h.5ucozq4kvv7o we would need the permission from the owner in order to include it here as BSD...

Reply:
Regrettably, I didn't realize that this might not be allowed. I'll contact the original author to ask for permission.

Reply:
I am glad to hear that my grammar is useful in this context. You have my permission.

Reply:
That is great to hear, thank you very much!